Pcap++/header/PcapLiveDevice.h

```diff
@@ -406,17 +406,39 @@ namespace pcpp
 		 */
 		bool captureActive();
 
+		/**
+		 * Checks whether the packetPayloadLength is larger than the device MTU. Logs an error if check fails
+		 * @param[in] packetPayloadLength The length of the IP layer of the packet
+		 * @return True if the packetPayloadLength is less than or equal to the device MTU
+		 */
+		bool doMtuCheck(int packetPayloadLength);
+
 		/**
 		 * Send a RawPacket to the network
 		 * @param[in] rawPacket A reference to the raw packet to send. This method treats the raw packet as read-only, it doesn't change anything
 		 * in it
+		 * @param[in] checkMtu Whether the length of the packet's payload should be checked against the MTU. If enabled this comes with a small performance penalty.
```

Comment:
I'd also mention that the default value is `false`

Reply:
Good point. Will add this.